### ISSUE TITLE
CLI wallet readline support

### ISSUE THREAD

**Issue by jagerman** [loki-project/loki-core#1361]
We use readline (at least for non-static linux builds), and it would be nice to optionally incorporate readline's history support.  We'd still want to encrypt the history file with the wallet password, and would want it to be optional, but it would be great (especially for testing) to have persistent history.

**Comment by darcys22** [comment]
Would you do this after every command is executed?

Such as Encrypt(command) -> prepend to history file

**Comment by jagerman** [comment]
No, just when saving to disk when you exit, or perhaps when you explicitly `save` (basically whenever it encrypts and writes the wallet cache to disk).
